helper_test.go

```diff
@@ -1177,6 +1201,47 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+// Attempts to upgrade an organization to the business plan. Requires a user token with admin access.
+func upgradeOrganizationSubscription(t *testing.T, client *Client, organization *Organization) {
```

Comment:
I wonder if this should do a t.Fatal if enterpriseEnabled() in order to reinforce that this doesn't work in TFE.